content/applications/inventory_and_mrp/manufacturing/basic_setup/bill_configuration.rst

```diff
@@ -2,98 +2,162 @@
+.. image:: bill_configuration/enable-work-orders.png
    :align: center
-   :alt: Product Variants in the Bill of Materials.
+   :alt: "Work Orders" feature in the Settings page.
 
-Set up operations
-=================
+Next, navigate to the |BOM| by going to :menuselection:`Manufacturing app --> Products --> Bill of
+Materials` and selecting the desired |BOM|. To add a new operation, go to the :guilabel:`Operations`
+tab, and click :guilabel:`Add a line`.
+
+Doing so opens the :guilabel:`Create Operations` pop-up window, where the various fields of the
+operation are configured:
+
+- :guilabel:`Operation`: name of the operation.
+- :guilabel:`Work Center`: select existing locations to perform the operation, or create a new work
+  center by typing the name and selecting the :guilabel:`Create...` option.
+- :guilabel:`Apply on Variants`: specify if this operation is only available for certain product
+  variants. If the operation applies to all product variants, leave this field blank.
+
+.. seealso::
+   :doc:`Configuring BoMs for product variants <../advanced_configuration/product_variants>`
+
+- :guilabel:`Duration Computation`: choose how time spent on the operation is tracked. Opt for
+  :guilabel:`Compute based on tracked time` use the operation's time tracker or :guilabel:`Set
+  duration manually` if operators can record and modify time themselves.
+- :guilabel:`Default Duration`: estimated amount of time to complete the operation; used for
+  `planning manufacturing orders <https://www.youtube.com/watch?v=TK55jIq00pc>`_ and determining
+  `work center availability <https://www.youtube.com/watch?v=3YwFlD97Bio>`_.
+- :guilabel:`Company`: specify the company the |BOM| is available in.
+
+Include operation details in the :guilabel:`Work Sheet` tab. Choose :guilabel:`PDF` to attach a file
+or :guilabel:`Google Slide` to share a link. Select :guilabel:`Text` to type instructions in the
+:guilabel:`Description` text field.
 
-Add an :guilabel:`Operation` to a :abbr:`BoM (Bill of Materials)` to specify instructions for
-production and register time spent on an operation. To use this feature, first enable the
-:guilabel:`Work Orders` feature in :menuselection:`Manufacturing --> Configuration --> Settings -->
-Operations`.
+.. image:: bill_configuration/create-operations.png
+   :align: center
+   :alt: Fill out the Create Operations pop-up window.
 
-Then, when creating a new :abbr:`BoM (Bill of Materials)`, click on the :guilabel:`Operations` tab
-and click :guilabel:`Add a line` to add a new operation. In the :guilabel:`Create Operations` box,
-give the operation a name, specify the :guilabel:`Work Center` and duration settings. Like
-components, Odoo gives the option to specify a product variant in the :guilabel:`Apply on Variants`
-field so the operation only applies to that variant. Finally, click :guilabel:`Save & Close`.
+Finally, click :guilabel:`Save & Close` to close the pop-up window. To add more operations, click
```

Comment:
@StraubCreative and @Felicious it is not necessary to escape ampersands in RST (unless it is in an HTML context like using the :raw: role/directive). In cases where it is used in paragraphs and plain text roles like :guilabel:s, it does not need to be escaped since it is treated as just a plain text special character (just like $, %, #, etc), and it is not considered an [inline markup character](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#inline-markup) either.

Example on the context:
<img width="232" alt="Screenshot 2024-05-17 at 11 02 04 AM" src="https://github.com/odoo/documentation/assets/36018073/cb3587ca-ee02-4467-9893-fbb1d9c1c211">

This is the output when rendered:
<img width="202" alt="Screenshot 2024-05-17 at 11 01 39 AM" src="https://github.com/odoo/documentation/assets/36018073/7b2e0a7c-765f-4626-8e4e-560582dd077a">

You can see the unicode character is only rendered in the HTML context, since the ampersand in that context indicates the beginning of an entity reference or numeric character reference. 

All that to say, escaping ampersands in plain text doesn't hurt– its just not necessary!